libfwupd/fwupd-remote.c

```diff
@@ -388,6 +388,11 @@ fwupd_remote_load_from_filename (FwupdRemote *self,
 				     keyring_kind);
 			return FALSE;
 		}
+		if (priv->keyring_kind == FWUPD_KEYRING_KIND_GPG ||
+		    priv->keyring_kind == FWUPD_KEYRING_KIND_PKCS7) {
+			g_debug ("converting Keyring value to Jcat");
```

Comment:
Maybe it's worth mentioning this as a warning that there is an old conf file in use.

Reply:
I don't think I can; the LVFS has to generate `Keyring=gpg` for remotes so it supports all fwupd versions.

Reply:
But you don't have to look at that one; you could instead try to find jcat ones specifically and ignore others.

Reply:
Not sure I follow, sorry.

Reply:
Whoops.  I was looking at the wrong function here, my mistake.